Patches/AnimalCollabProj/AnimalCollabProj_CE_Patch_Races_Animal_Horse.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>AnimalCollabProj</modName>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[
+					defName="ACPHorse" or 
+					defName="ACPWoolyHorse"
+				]/statBases</xpath>
+				<value>
```

Comment:
Vanilla horse for reference. Given the differences in the various breeds of horses, your call if you think it's appropriate to adjust.
```
			<MoveSpeed>5</MoveSpeed>
			<MeleeDodgeChance>0.11</MeleeDodgeChance>
			<MeleeCritChance>0.30</MeleeCritChance>
			<MeleeParryChance>0.19</MeleeParryChance>
```

Reply:
Probably not much difference between both breeds, so I'm setting them to vanilla values as well.

Fixed in 087aad7